setup.py

```diff
@@ -79,6 +79,8 @@
         data_qld_google_analytics=ckanext.data_qld.google_analytics.plugin:GoogleAnalyticsPlugin
         data_qld_reporting=ckanext.data_qld.plugin:PlaceholderPlugin
 
+        data_qld_test=ckanext.data_qld.test_plugin:DataQldTestPlugin
```

Comment:
please move data_qld_test into its own repo/name space. 

We are in the process of cleaning up entry points to only have 1 per extension as it causes tight coupling on our deployment/devops pipeline on which version has which plugin names we need to enable etc.

Reply:
Will be fixed and pushed by Alex shortly.

Reply:
I'm not going to move it to the separate extension, until we are fine with using this approach.